### PR TITLE
subgraph/timelock multi operation

### PR DESCRIPTION
**Pull request by hujw77**
- support multi operation
- clear timestamp when cancel proposal

### CODE REVIEW

File: packages/subgraph/TimeLock/schema.graphql [hunk 1]
```diff
@@ -1,11 +1,24 @@
-type ProposalItem @entity {
+enum Status {
+    Pending
+    Ready
```
Comment (xiaoch05) [inline packages/subgraph/TimeLock/schema.graphql:3]: suggest change `Pending` and `Ready` status to `WaitingExecuted` status.

File: packages/subgraph/TimeLock/src/mapping.ts [hunk 2]
```diff
@@ -3,48 +3,48 @@ import {
...
-    if (entity == null) {
-      return
-    }
-  entity.status = 3
+  let entity = Proposal.load(event.params.id.toHex())
+  if (entity == null) {
+    return
+  }
+  entity.status = 'Cancelled'
+  entity.timestamp = BigInt.zero()
```
Comment (xiaoch05) [inline packages/subgraph/TimeLock/src/mapping.ts:48]: is it better to set timestamp to block.timestamp?

File: packages/subgraph/TimeLock/schema.graphql (hunk 1 above)
Comment (hujw77) [reply packages/subgraph/TimeLock/schema.graphql:3]: https://github.com/darwinia-network/wormhole-dao#proposal-lifecycle
`Pending` status could not execute the proposal

File: packages/subgraph/TimeLock/src/mapping.ts (hunk 2 above)
Comment (hujw77) [reply packages/subgraph/TimeLock/src/mapping.ts:48]: https://github.com/darwinia-network/darwinia-bridge-sol/blob/dc35bc7574d2c5f383506b2fa51539531046688c/contracts/dao/flat/solc-0.8.6/TimeLock.f.sol#L716
I think `set it to 0` is the more in line with the contract design.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**